spec/components/email/receiver_spec.rb

```diff
@@ -462,6 +468,16 @@ def process(email_name)
       expect { process(:new_user) }.to change(Topic, :count)
     end
 
+    it "adds the 'elided' part of the original message when always_show_trimmed_content is enabled" do
+      SiteSetting.always_show_trimmed_content = true
+
+      user = Fabricate(:user, email: "existing@bar.com", trust_level: SiteSetting.email_in_min_trust)
+      expect { process(:forwarded_email_to_category) }.to change(Topic, :count) # Topic created
```

Comment:
Let's follow existing conventions in our code base and use `.to change { Topic.count }`